SECURITY.md

```diff
@@ -12,3 +12,12 @@ You may expect a response within 48 hours.
 
 For extended support or a stronger guarantee,
 please reach out to the maintainers to discuss a support agreement.
+
+## Escalation
+
+If you do not receive an acknowledgement of your report within 6 business days, 
+or if you cannot find a private security contact for the project, you may escalate
+to the OpenJS Foundation CNA at `security@lists.openjsf.org`.
```

Comment:
Given how the reporting method is defined immediately above this text, the mention about not finding it is not appropriate.

```suggestion
you may escalate to the OpenJS Foundation CNA at `security@lists.openjsf.org`.
```